src/deployment/azuredeploy.json

```diff
@@ -67,7 +67,8 @@
         "Storage Account Contributor": "17d1049b-9a84-46fb-8f53-869881c3d3ab",
         "Virtual Machine Contributor": "9980e02c-c2be-4d73-94e8-173b1dc7cf3c",
         "Storage Blob Data Reader": "2a2b9908-6ea1-4ae2-8e65-a410df84e7d1",
-        "keyVaultName": "[concat('of-kv-', uniquestring(resourceGroup().id))]"
+        "keyVaultName": "[concat('of-kv-', uniquestring(resourceGroup().id))]",
+        "fuzz-blob-topic-name": "[concat('fuzz-blob-topic-', uniquestring(resourceGroup().id))]"
```

Comment:
What's the purpose of the "fuzz-blob-topic-name"? It's strikes me as fairly generic and I'm wondering if its the right name.

Reply:
It is the topic created to gather the notification of the blobs in the fuzz storage account.
We subscribe to these events to trigger the bug filling function. 
I am open to suggestions on the name